app/js/features/escrow/constants.js

```diff
@@ -13,3 +13,7 @@ export const RELEASE_ESCROW_FAILED = 'RELEASE_ESCROW_FAILED';
 export const CANCEL_ESCROW = 'CANCEL_ESCROW';
 export const CANCEL_ESCROW_SUCCEEDED = 'CANCEL_ESCROW_SUCCEEDED';
 export const CANCEL_ESCROW_FAILED = 'CANCEL_ESCROW_FAILED';
+
+export const RATE_TRANSACTION = 'RATE_TRANSACTION';
```

Comment:
I have the feeling we miss the rating feature?

Reply:
Well, the rating in part of the escrow (one rating per escrow).
We could move it to another feature, but for now, it would only contain 2 actions, so maybe not necessary now?

Reply:
up to you, you are best judges :+1: